cli/src/main/kotlin/OrtMain.kt

```diff
@@ -123,7 +124,8 @@ class OrtMain : CliktCommand(ORT_NAME) {
             helpFormatter = { MordantHelpFormatter(context = it, REQUIRED_OPTION_MARKER, showDefaultValues = true) }
         }
 
-        subcommands(OrtCommand.ALL.values)
+        // Commands are not configurable so we can pass an empty PluginConfig here.
```

Comment:
Nit: I guess we should use passive voice here, too. Like "Pass an empty PluginConfig here as commands are not configurable."

Reply:
Should we make the code self-explanatory to remove the need for this comment? 
I guess the comment would be repeated over and over again. 

We can put a constant `PluginConfig.EMPTY`, or similar, which would also avoid unnecessary instance creation.

Reply:
> I guess the comment would be repeated over and over again.

Do you mean for other plugin types? I'm not sure about that, as commands are really special in this regard.

However, I was also wondering whether we should allow `factory.create()` to take no arguments for this case instead of introducing `PluginConfig.EMPTY`.

Reply:
I meant writing here: `subcommands(OrtCommandFactory.ALL.map { (_, factory) -> factory.create(PluginConfig.EMPTY) })` 

And drop the code comment.

Reply:
Understood. By my question was: *Where* do you potentiality see the comment to "repeated over and over again"? In upcoming plugin migrations?

Reply:
I didn't have any specific callers in mind. I just thought that this comment would need to be repeated for each `create()` call which passes `PluginConfig()` for consistency.

Reply:
I have updated the comment to not use passive voice.

> However, I was also wondering whether we should allow factory.create() to take no arguments for this case instead of introducing PluginConfig.EMPTY.

I'd rather not introduce that option before all plugins are migrated to avoid forgetting to pass the config.

> I meant writing here: subcommands(OrtCommandFactory.ALL.map { (_, factory) -> factory.create(PluginConfig.EMPTY) })
> And drop the code comment.

That code still does not explain why it is ok to pass an empty config so I would keep the comment anyway.